src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -53,6 +53,7 @@ import { visualizationLoader } from 'ui/visualize/loader/visualization_loader';
 import { recentlyAccessed } from 'ui/persisted_log';
 import { getDocLink } from 'ui/documentation_links';
 import '../components/fetch_error';
+import { getPainlessError } from './get_painless_error';
```

Comment:
__question:__ I still see "old" implementation of `getPainlessError`, is it just leftover you we still need it for some reason?